src/icons/Coin.tsx

```diff
@@ -0,0 +1,33 @@
+import CommonIconsProps from 'icons/CommonIconsProps'
+import classnames, { stroke, strokeWidth } from 'classnames/tailwind'
+
+export default function ({ inheritStrokeColor }: CommonIconsProps) {
+  const commonPathStyles = classnames(
```

Comment:
Placing `classnames` inside the function saves us a little time, we don't need to do:
```TS
const commonPathStyles = (inherit?: boolean) => ...

className={commonPathStyles(inheritStrokeColor)}
```
I think it's a good practice to place them inside the function component :)